lib/mix/lib/mix/project.ex

```diff
@@ -249,13 +256,16 @@ defmodule Mix.Project do
       #=> %{my_app1: "apps/my_app1", my_app2: "apps/my_app2"}
 
   """
-  @spec apps_paths() :: %{optional(atom) => Path.t} | nil
+  @spec apps_paths() :: %{optional(atom) => Path.t()} | nil
   def apps_paths(config \\ config()) do
     if apps_path = config[:apps_path] do
-      key = {:apps_paths, Mix.Project.get!}
+      key = {:apps_paths, Mix.Project.get!()}
+
       Mix.ProjectStack.read_cache(key) ||
-        Mix.ProjectStack.write_cache(key,
-          config[:apps] |> umbrella_apps(apps_path) |> to_apps_paths(apps_path))
+        Mix.ProjectStack.write_cache(
+          key,
+          config[:apps] |> umbrella_apps(apps_path) |> to_apps_paths(apps_path)
```

Comment:
This pipe seems questionable, but refactoring this line would require to do more complicated code structure as I understand (it is not just extracting a variable, but wrapping it all in if/else)